test/parallel/test-fs-write-stream-file-handle.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+const tmpdir = require('../common/tmpdir');
+const file = path.join(tmpdir.path, '/write_stream_filehandle_test.txt');
+const input = 'hello world';
+
+tmpdir.refresh();
+
+fs.promises.open(file, 'w+').then((handle) => {
```

Comment:
`common.mustCall()` for this one, and ditto for all the tests in the file above?